ui/src/shared/utils/featureFlag.ts

```diff
@@ -63,36 +63,21 @@ export const FeatureFlag: FunctionComponent<{
   return children as any
 }
 
+export const getUserFlags = function getUserFlags() {
+  const flagKeys = CLOUD ? Object.keys(CLOUD_FLAGS) : Object.keys(OSS_FLAGS)
+
+  const flags = {}
+  flagKeys.forEach(key => {
+    flags[key] = isFlagEnabled(key)
+  })
+
+  return flags
+}
+
 /* eslint-disable no-console */
 const list = () => {
   console.log('Currently Available Feature Flags')
-  if (CLOUD) {
-    console.table(
-      Object.keys(CLOUD_FLAGS)
-        .map(k => [k, isFlagEnabled(k)])
-        .reduce((prev, curr) => {
-          if (typeof curr[0] === 'boolean') {
-            return prev
-          }
-
-          prev[curr[0]] = curr[1]
-          return prev
-        }, {})
-    )
-  } else {
-    console.table(
-      Object.keys(OSS_FLAGS)
-        .map(k => [k, isFlagEnabled(k)])
-        .reduce((prev, curr) => {
-          if (typeof curr[0] === 'boolean') {
-            return prev
-          }
-
-          prev[curr[0]] = curr[1]
-          return prev
-        }, {})
-    )
-  }
+  console.table(getUserFlags())
```

Comment:
as far as I could tell in my local testing, `getUserFlags` has the same return signature as 

```ts
 Object.keys(CLOUD_FLAGS || OSS_FLAGS)
        .map(k => [k, isFlagEnabled(k)])
        .reduce((prev, curr) => {
          if (typeof curr[0] === 'boolean') {
            return prev
          }

          prev[curr[0]] = curr[1]
          return prev
        }, {})
```